jetstack/jsctl
# Parameterize cainjector image, add latest operator version

This PR:

- ensures that a registry passed by `--registry` flag to `operator deploy` command is also used for cainjector image

- by default pulls cainjector image from JSE GCR, not quay.io (because we want JSE GCR to be the single place where enterprise customers can consume images from)

- adds v0.0.1-alpha.17 version of js-operator


To test this
1. Set up a cluster with agent
2. Run `go run main.go operator deploy --auto-registry-credentials`
3. Observe that cainjector and js-operator deployments become ready and the operator is at version v0.0.1-alpha.17
